src/types/Expr.d.ts

```diff
@@ -3,5 +3,6 @@ export default class Expr {
 
   readonly _isFaunaExpr?: boolean
   toFQL(): string
+  toJSON(): string
```

Comment:
```suggestion
  toJSON(): Record<string, any>
```

Reply:
The `toJSON()` metho returns the underlying object. In that, "JSON" is a little misleading since it's not a string. JS makes it trivial to convert an object to a JSON string, so it's (too) easy to conflate the two.